.travis.yml

```diff
@@ -10,6 +10,8 @@ env:
   - TOX_ENV=py32_dj18
   - TOX_ENV=py33_dj18
   - TOX_ENV=py34_dj18
+  - TOX_ENV=py27_dj19b1
+  - TOX_ENV=py34_dj19b1
```

Comment:
I forgot that it was because pyenv wasn't listing it for me, but that was because I had an old version of pyenv. Will add.